test/unit/api_spec.js

```diff
@@ -2682,6 +2682,44 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       await loadingTask.destroy();
     });
 
+    it("check that a chunk is pushed when font is restored", async function () {
+      const loadingTask = getDocument(buildGetDocumentParams("issue14755.pdf"));
+      const pdfDoc = await loadingTask.promise;
+      const pdfPage = await pdfDoc.getPage(1);
+      const { items } = await pdfPage.getTextContent({
+        disableNormalization: true,
+      });
+      expect(items).toEqual([
+        jasmine.objectContaining({
+          str: "ABC",
+          dir: "ltr",
+          width: 20.56,
+          height: 10,
+          transform: [10, 0, 0, 10, 100, 100],
+          hasEOL: false,
+        }),
```

Comment:
Here, and below, I don't *think* that you need to use `jasmine.objectContaining` if you're comparing against a "complete" object. Hence the following ought to work as well:
```suggestion
        {
          str: "ABC",
          dir: "ltr",
          width: 20.56,
          height: 10,
          transform: [10, 0, 0, 10, 100, 100],
          hasEOL: false,
        },
```